temporalio/contrib/opentelemetry.py

```diff
@@ -244,6 +244,28 @@ async def signal_workflow(
         ):
             return await super().signal_workflow(input)
 
+    async def start_workflow_update(
+        self, input: temporalio.client.UpdateWorkflowInput
+    ) -> temporalio.client.WorkflowUpdateHandle:
+        with self.root._start_as_current_span(
+            f"StartWorkflowUpdate:{input.update}",
+            attributes={"temporalWorkflowID": input.id},
+            input=input,
+            kind=opentelemetry.trace.SpanKind.CLIENT,
+        ):
+            return await super().start_workflow_update(input)
+
+    async def poll_workflow_update(
```

Comment:
No concept of headers on polling therefore no value in serializing a new span here (and no inbound call on the worker side to take it). Can probably just remove this method impl on this class.